tests/url.sil

```diff
@@ -0,0 +1,10 @@
+\begin{document}
```

Comment:
In tests, papersize a6 for small tests, @alerque once told me (default is a4). Quoting his words from another review:
> Another test detail is I really prefer tests to use small pages where possible. I sometimes generate PDFs of everything and make a montage of them for review. Big A4 pages waste a lot of space and make these hard to review. A6 for singe pages tests and A7 for multi-page tests are preferred if possible. This way the default font sizes make the tests easier to review en masse.